PoliceUK/Entities/Neighbourhood/NeighbourhoodLocation.cs

```diff
@@ -11,7 +11,11 @@ public class NeighbourhoodLocation
         [DataMember(Name = "name")]
         public string Name { get; set; }
 
-        public IGeoposition Position { get; set; } // XXX Derive from same level lat/lng properties
+        [DataMember(Name = "latitude")]
```

Comment:
Using the basic binding for now, happy to change this later if needed to a (I)GeoPosition, but this achieves the purpose for now.